opm/autodiff/SimulatorFullyImplicitBlackoilEbos.hpp

```diff
@@ -328,33 +328,46 @@ class SimulatorFullyImplicitBlackoilEbos
             std::vector<std::vector<double>> currentFluidInPlace;
             currentFluidInPlace = solver->computeFluidInPlace(fipnum);
             std::vector<double> currentFluidInPlaceTotals = FIPTotals(currentFluidInPlace, state);
+			      const std::string version = moduleVersionName();
 
             FIPUnitConvert(eclState().getUnits(), currentFluidInPlace);
             FIPUnitConvert(eclState().getUnits(), currentFluidInPlaceTotals);
 
+            if ( output_writer_.output() ) {
+                if ( output_writer_.isIORank() )
+                {
+                    stepReport.reportParam(tstep_os);
+                }
+            }
+
+            // Increment timer, remember well state.
+            ++timer;
+			
             if (terminal_output_ )
             {
+                std::ostringstream ss;
+                boost::posix_time::time_facet* facet = new boost::posix_time::time_facet("%d %b %Y");
+                ss.imbue(std::locale(std::locale::classic(), facet));
+                ss <<"\n                              **************************************************************************   "
```

Comment:
Always put spaces around binary operators such as `<<` (some exceptions ok, if improving readability, such as around `*` in some long expressions).